assets/newsletter-section.css

```
@@ -48,3 +91,12 @@
     margin: 0 0 0 1rem;
   }
 }
+@media screen and (max-width: 750px) {
+  .newsletter__wrapper-new {
```

```suggestion
  .newsletter__wrapper{
```

I want to remove all instances of `-new` on these classes, let's modify what is here to match our design.